torchvision/models/detection/anchor_utils.py

```diff
@@ -100,7 +100,10 @@ def grid_anchors(self, grid_sizes, strides):
         anchors = []
         cell_anchors = self.cell_anchors
         assert cell_anchors is not None
-        assert len(grid_sizes) == len(strides) == len(cell_anchors)
+
+        if len(grid_sizes) != len(strides) != len(cell_anchors):
+            raise ValueError('''Achors should be Tuple[Tuple[int]] because each feature
+                                map could potentially have different sizes and aspect ratios ''')
```

Comment:
Can you use a single `"`, and not break the line as currently done, as it introduces artifacts on the print?
Also, can you make the error message a bit more descriptive, as you can have `Tuple[Tuple[int]]` being passed but still have the same problem. The underlying issue is that there needs to be a match between the number of feature maps passed and the number of sizes / aspect ratios specified.